CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+afr_module(INTERNAL)
```

Comment:
Is this just a bulk standard cmakelists.txt file?  If it is in the root directory within the source of truth repo it can't have anything Amazon specific in the file.